ansible/roles/docker/templates/docker.service

```diff
@@ -8,7 +8,7 @@ Requires=docker.socket
 [Service]
 Type=notify
 # FIXME: Find what port should be used in cluster-advertise when not using swarm
-ExecStart=/usr/bin/docker daemon -H fd:// --cluster-store="etcd://{% for host in groups['etcd_masters'] %}{{ hostvars[host][private_interface]['ipv4']['address'] }}:2379{% if not loop.last %},{% endif %}{% endfor %}" --cluster-advertise={{ hostvars[inventory_hostname][private_interface].ipv4.address }}:0
+ExecStart=/usr/bin/docker daemon -H tcp://0.0.0.0:2375 -H fd:// --cluster-store="etcd://{% for host in groups['etcd_masters'] %}{{ hostvars[host][private_interface]['ipv4']['address'] }}:2379{% if not loop.last %},{% endif %}{% endfor %}" --cluster-advertise={{ hostvars[inventory_hostname][private_interface].ipv4.address }}:0
```

Comment:
The ports here can be replaced with variables. Also instead of `:0` in `cluster_advertise` lets use the docker port